src/app/screens/LNURLPay.tsx

```diff
@@ -212,42 +205,96 @@ function LNURLPay(props: Props) {
     return elements;
   }
 
+  function renderSuccessAction() {
+    if (successAction) {
```

Comment:
maybe `if (!successAction) { return; }` to prevent the nesting?

Reply:
Will remove the check and make TS assume the value is present.
As renderSuccessAction is only called when successAction exists.